scripts/launch/demo-with-new-wallet-api.sh

```diff
@@ -2,4 +2,4 @@
 
 base=$(dirname "$0")
 
-WALLET_EXTRA_ARGS="--new-wallet" WALLET_EXE_NAME='cardano-node' WALLET_TEST=1 "$base"/demo.sh "$@"
+WALLET_EXE_NAME='cardano-node' WALLET_TEST=1 "$base"/demo.sh "$@"
```

Comment:
For some reason having the `WALLET_EXTRA_ARGS` was causing the `--new-wallet` option to be added twice in the final launch script, causing the demo cluster not to run when invoked via `./scripts/launch/demo-with-new-wallet-api.sh`.